tests/API/JsonDeserializeTest.php

```diff
@@ -57,6 +59,8 @@ public static function init()
             '"range":true,"or":false,"type":"Range_Date","sort":"Value_Ascending","refinements":[' .
             self::$JSON_REFINEMENT_RANGE . ',' . self::$JSON_REFINEMENT_VALUE .
             '],"metadata":[' . self::$JSON_METADATA . ']}';
+
+        self::$JSON_RESTRICT_NAVIGATION = '{"name":"categories","count":2}';
```

Comment:
because this doesn't rely on other values, there's no reason to set it in init(). Should just initialize it on definition at the top.